apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-groups/vo-groups.component.html

```diff
@@ -11,18 +11,20 @@ <h1 class="page-subtitle">{{'VO_DETAIL.GROUPS.TITLE' | translate}}</h1>
     [placeholder]="'VO_DETAIL.GROUPS.FILTER'"
     (filter)="applyFilter($event)">
   </app-debounce-filter>
-  <mat-checkbox color="primary" class="ml-3" [(ngModel)]="showGroupList" (change)="removeAllGroups()" #checkbox>
-    {{'VO_DETAIL.GROUPS.LIST_VIEW' | translate}}
-  </mat-checkbox>
+  <mat-slide-toggle #toggle (change)="removeAllGroups()" [(ngModel)]="showGroupList" class="ml-3" color="primary"
```

Comment:
When you want to add some space between two elements, prefer to set the right margin of the element on the left. If you would do it this way, then when it is displayed on small devices where the whole content cannot fit on a single line, all elements will be nicely aligned on the left side.
![before](https://user-images.githubusercontent.com/26108901/88786358-b75d5900-d192-11ea-8071-5610f8083e54.png)
![after (1)](https://user-images.githubusercontent.com/26108901/88786475-d825ae80-d192-11ea-8494-aa598641784f.png)